Ombi.UI/Modules/SearchModule.cs

```diff
@@ -222,6 +229,36 @@ public class SearchModule : BaseAuthModule
             return await ProcessMovies(MovieSearchType.Search, searchTerm);
         }
 
+        private async Task<Response> SearchActor(string searchTerm)
+        {
+            var movies = TransformMovieListToMovieResultList(await MovieApi.SearchActor(searchTerm).ConfigureAwait(false));
+            return await TransformMovieResultsToResponse(movies);
+        }
+
+        private async Task<Response> SearchActor(string searchTerm, bool filterExisting)
+        {
+            var movies = TransformMovieListToMovieResultList(await MovieApi.SearchActor(searchTerm, AlreadyAvailable).ConfigureAwait(false));
+            return await TransformMovieResultsToResponse(movies);
+        }
+
+        private async Task<bool> AlreadyAvailable(int id, string title, string year)
+        {
+            await Task.Yield();
+            return IsMovieInCache(id, String.Empty) || PlexChecker.IsMovieAvailable(plexMovies(), title, year);
+        }
+
+        private IEnumerable<PlexContent> plexMovies()
```

Comment:
Can definitely remove this, I just thought it would be neat since I feel the use case is a bit different than searching straight for a movie.  Like, when searching for a movie, I have a title in mind and I want to find it.

When searching for a movie, I'm probably more thinking, "What movies is So-and-so in that I don't have yet?"

If we only ever show the first 10, it loses some longer term value.  I suppose the **real** problem is that we are only showing 10 and don't offer a way to page through all results.  So this was an attempt to get more than the first 10.

If you think it's too different, I can stash this stuff aside for later consideration.